gazebo/sensors/DepthCameraSensor_TEST.cc

```diff
@@ -193,6 +193,75 @@ TEST_F(DepthCameraReflectanceSensor_TEST, CreateDepthCamera)
     delete [] g_reflectanceBuffer;
 }
 
+using namespace gazebo;
+class DepthCameraSensor_reflectance_topic_TEST : public ServerFixture
+{
+};
+
+std::mutex g_reflectanceTopicMutex;
+bool g_reflectanceTopicReceivedMsg;
+boost::shared_ptr<msgs::PropagationGrid const> g_reflectanceTopicGridMsg;
+
+/////////////////////////////////////////////////
+/// \brief Callback executed for every propagation grid message received
+void TxMsg(const ConstPropagationGridPtr &_msg)
+{
+  std::lock_guard<std::mutex> lock(g_reflectanceTopicMutex);
+  // Just copy the message
+  g_reflectanceTopicGridMsg = _msg;
+  g_reflectanceTopicReceivedMsg = true;
+}
```

Comment:
You should include this variables and function inside the class `DepthCameraSensor_reflectance_topic_TEST`